### ISSUE TITLE
SCSS File Separation 

### ISSUE THREAD

**Issue by NeilWBruce** [needim/noty#389]
Would it be possible in the noty.scss file to either remove the themes imports or move the standard noty css to it's on scss file, this way we could compile the noty.css without the need to take in all the themes, we could choose which ones we want or use our own custom ones.

**Comment by needim** [comment]
Yeah, but it means another network request for very small file. Maybe we can separate later if we have more themes.

**Comment by claviska** [comment]
I second this request. I realize a lot of people will just copy and paste or use a CDN, but devs using the SCSS files will have unnecessary theme bloat.

I'd also suggest prefixing [these vars](https://github.com/needim/noty/blob/master/src/noty.scss#L1-L3) with `$noty-` to prevent collisions when including Noty's styles into another project. 😄

**Comment by kshnurov** [comment]
If you're using only 1 theme (as most do) with latest noty version (3.1.4), 2/3 of CSS will stay unused. It's about 20KB (or 1.3KB gzipped). @needim, please, move imports to separate file.

**Comment by needim** [comment]
Ok, I will separate themes for next release. Probably next week. 👍

**Comment by needim** [comment]
Hi guys, With v3.2.0-beta now we have separate theme CSS files. Also, variables are now prefixed with "$noty-". Are we okay? 😅

**Comment by claviska** [comment]
Beautiful. I'll update Noty my project tomorrow and report back if I run into any issues. 👏
